scripts/generate-markdown-toc.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * @fileoverview This script generates a table of contents (TOC) for a markdown file.
+ * It reads the markdown file, generates the TOC, and updates the file with the TOC.
+ *
+ * Arguments:
+ * - The file name for which the TOC should be generated.
+ *
+ * @example node scripts/generate-markdown-toc.js README.md
+ * @example node scripts/generate-markdown-toc.js docs/README.md
+ *
+ * Usage:
+ * Add the following placeholders in the markdown file where the TOC should be placed.
+ * <!-- TOS:START - Do not remove or modify this section -->
+ *
+ * <!-- TOS:END -->
+ */
+
+const fs = require("fs");
+const toc = require("markdown-toc");
+
+// eslint-disable-next-line no-console
```

Comment:
We don't need this comment as eslint only catch occurrences of `console.*`.

```suggestion
```

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/6560/commits/16a9243b7b5014bf57de86cf4e76f53f0be01cd1